ideascube/serveradmin/tests/test_catalog.py

```diff
@@ -130,6 +141,31 @@ def test_remote_to_file(tmpdir):
         'url: http://foo.fr/catalog.yml']
 
 
+def test_remote_to_file_utf8(tmpdir):
+    from ideascube.serveradmin.catalog import Remote
+
+    path = tmpdir.join('foo.yml')
+
+    remote = Remote(
+        'bibliothèque', 'Le contenu de la bibliothèque',
+        'http://foo.fr/catalog.yml')
+    remote.to_file(path.strpath)
+
+    lines = path.readlines(cr=False)
```

Comment:
Huh. I'm surprised this works, indeed. I guess `.readlines()` must be doing something magic... 😕 

I'm not super comfortable witht hat, so I've replaced it by a more explicit `read_text('utf-8').split('\n')`.